### ISSUE TITLE
Remove the Actions column from the Recent Jobs page

### ISSUE THREAD

**Issue by ITAYC0HEN** [CERT-Polska/mquery#24]
**Description**
It seems like the Actions column in `/recent` is used only for displaying the Cancel button for active jobs. While this is nice, it takes unnecessary places because most of the time there are one or none active jobs. 

![image](https://user-images.githubusercontent.com/20182642/78381007-658a3500-75dd-11ea-9771-4e669e15d9f8.png)

My suggestion is to add the cancel button next to the active scan  or in the Status cell.

In virus total, they put it in the column of the Delete task buttons (see #21)

![image](https://user-images.githubusercontent.com/20182642/78380834-2c51c500-75dd-11ea-9844-d0fce967340e.png)

**Comment by CodingApps** [comment]
I'd like to assist with this.

**Comment by msm-code** [comment]
Hi,

great, thanks for suggesting. To start working on this issue you should clone the repository, and then `docker-compose build` and `docker-compos up` in the checked out directory.

The file that needs to be modified is `SearchJobs.js`. I'm not sure how exactly - maybe merge the "actions" with  "progress" and present it like this:

![ss](https://user-images.githubusercontent.com/7026881/78501882-6e9a1400-774d-11ea-96b2-dc9e3f6a4003.png)

This is the HTML I've tried for this demo:

```html
<div>
   processing
  <button type="button" class="close" data-toggle="tooltip" title="cancel">
    <span>×</span>
  </button>
</div>
```

(remember to run your changes via prettier (https://prettier.io/) before submitting PR)

**Comment by msm-code** [comment]
Fixed with #129
